.scripts/_dependencies.sh

```
@@ -59,7 +75,29 @@ function dependency() {
+      if $is_optional; then
+        # shellcheck disable=SC2154
+        echo "${cl_blue}auto-installing${cl_reset} missing optional dependency \`${cl_yellow}$tool_name${cl_reset}\`"
+      else
+        # shellcheck disable=SC2154
+        echo "${cl_blue}auto-installing${cl_reset} missing dependency \`${cl_yellow}$tool_name${cl_reset}\`"
+      fi
+
+      if eval $tool_fallback; then
+        # Verify the tool is now available
+        if command -v "$tool_name" >/dev/null 2>&1; then
+          echo "${cl_green}✓${cl_reset} Successfully installed \`$tool_name\`"
```

Shellcheck (0.11.0)

[warning] cl_green is referenced but not assigned.

(SC2154)